_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw.cpp

```diff
@@ -26,6 +26,23 @@
 
 namespace MfxHwVP9Encode
 {
+bool CheckTriStateOption(mfxU16 & opt);
+
+void SetDefaultForLowpower(VP9MfxVideoParam& par, eMFXHWType platform)
+{
+    CheckTriStateOption(par.mfx.LowPower);
+
+    if (par.mfx.LowPower == MFX_CODINGOPTION_UNKNOWN)
+    {
+#if (MFX_VERSION >= 1027)
+        if (platform >= MFX_HW_ICL)
```

Comment:
you may fall to static analysis warning for unused variable. You may fix it like this for example:
```
#if (MFX_VERSION >= 1027)
        if (platform >= MFX_HW_ICL)
            par.mfx.LowPower = MFX_CODINGOPTION_ON;
        else
#else
        (void)platform; // or std::ignore = platform;
#endif
```